TESTS_GUIDELINES.md

```diff
@@ -0,0 +1,54 @@
+# Guidelines for Creating and Executing Tests in Vanilla OS
+
+## Introduction
+
+This document serves as a reference for maintainers of Vanilla OS repositories in creating and managing testing procedures. The goal is to ensure the integrity, security, and functionality of components through consistent and well-structured tests, identifying potential issues before they are brought into production.
+
+It is important that all maintainers are familiar with and have understood the [Vanilla OS contribution guidelines](https://github.com/Vanilla-OS/.github/blob/main/CONTRIBUTING.md).
+
+## Responsibilities of Maintainers
+
+Maintainers are responsible for the continuous integration of tests into their projects. Each new feature or fix should be accompanied, where possible, by relevant tests that validate the changes and ensure that existing functionalities remain intact.
```

Comment:
What about "It is the responsibility of the maintainers to ensure the continuous integration (CI) of tests into their respective projects."?

Reply:
Yeah I like your proposal